src/vendor/automerge-wasm/index.d.ts

```diff
@@ -0,0 +1,313 @@
+export type Actor = string;
```

Comment:
Is this all just from upgrading automerge-wasm?

Reply:
I've just copied that over from trail-runner